File: Course_01/Java Language/Estrutura condicional/notas.java
package notas;

import java.util.Locale;
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double n1, n2, nf;
		
		System.out.print("Digite a primeira nota: ");
		n1 = sc.nextDouble();
		System.out.print("Digite a segunda nota: ");
		n2 = sc.nextDouble();
		
		nf = n1 + n2;
		System.out.println("NOTA FINAL = " + nf);
		if (nf < 60) {
			System.out.println("REPROVADO");
		}
		sc.close();
	}

}

File: Course_01/C++ Language/EstruturaCondicional/notas.cpp
#include <iostream>

using namespace std;

int main()
{
    double n1, n2, nf;

    cout << "Digite a primeira nota: ";
    cin >> n1;

    cout << "Digite a segunda nota: ";
    cin >> n2;

    nf = n1 + n2;

    cout << "NOTA FINAL = " << nf << endl;

    if (nf < 60) {
        cout << "REPROVADO";
    }


    return 0;
}
